wremnants/include/utils.h

```diff
@@ -188,6 +188,19 @@ bool hasMatchDR2(const float& eta, const float& phi, const Vec_f& vec_eta, const
 
 }
 
+RVec<Int_t> hasMatchDR2collWithSingle(const Vec_f &coll1_eta, const Vec_f &coll1_phi,
```

Comment:
I think this can also be removed together with the block in the histmaker

Reply:
Well, this is a general function that might be needed at some point, and utils.h seems the right place to have it